src/sagas/identity/identity.selectors.test.ts

```diff
@@ -0,0 +1,254 @@
+import { combineReducers, createStore, Store } from '@reduxjs/toolkit';
+import { StoreKeys } from '../store.keys';
+import { communitiesAdapter } from './../communities/communities.adapter';
+import {
+  communitiesReducer,
+  CommunitiesState,
+  Community,
+} from './../communities/communities.slice';
+import { identityAdapter } from './identity.adapter';
+import { identitySelectors } from './identity.selectors';
+import { Identity, identityReducer, IdentityState } from './identity.slice';
+
+describe('communitiesSelectors', () => {
```

Comment:
Is there a more readable description that would be appropriate here? What are we checking?